packages/checkbox-group/test/lion-checkbox-indeterminate.test.js

```diff
@@ -145,12 +158,15 @@ describe('<lion-checkbox-indeterminate>', () => {
     // Act
     elIndeterminate._inputNode.click();
     await elIndeterminate.updateComplete;
+    const { subCheckboxes } = getProtectedMembers(elIndeterminate);
 
     // Assert
-    expect(elIndeterminate?.hasAttribute('indeterminate')).to.be.false;
-    expect(elIndeterminate?._subCheckboxes[0].hasAttribute('checked')).to.be.true;
-    expect(elIndeterminate?._subCheckboxes[1].hasAttribute('checked')).to.be.true;
-    expect(elIndeterminate?._subCheckboxes[2].hasAttribute('checked')).to.be.true;
+    if (elIndeterminate) {
```

Comment:
I think the test should fail if `elIndeterminate` is null (is it possible to remove this if? (according to lin 154, TS shouldn't complain)